client/app/account/user/agreements/details/user-agreements-details.html

```diff
@@ -2,18 +2,13 @@
      class="module-useraccount-sections-agreements-container">
 
     <div class="page-header">
-        <h1 data-translate="user_agreement_details_title"></h1>
+        <h1 data-translate="user_agreements_title"></h1>
     </div>
 
     <div class="tab-content">
-        <a class="d-inline-block mb-5"
-           href="#/useraccount/agreements"
-           title="{{ 'user_agreement_details_return' | translate }}">
-            <span class="fa fa-chevron-left mr-2"
-                  aria-hidden="true">
-            </span>
-            <span data-translate="user_agreement_details_return"></span>
-        </a>
+        <oui-back-button data-href="#/useraccount/agreements"
```

Comment:
Why not using the state reference `"app.account.service.useraccount.agreements"` instead?

Reply:
Because it's not allowed ```data-sref="app.account.service.useraccount.agreements"``` is not allowed by the ovh-ui-angular